docs/adrs/add-auto-import-containerd.md

```diff
@@ -0,0 +1,54 @@
+# Easy way for auto adding images to k3s
+
+Date: 2024-10-2
+
+## Status
+
+Proposed
+
+## Context
+
+Since the feature for embedded registry, the users appeared with a question about having to manually import images, specially in edge environments.
```

Comment:
Why would users have to manually import images? Aren't they automatically pulled by containerd once the pod is scheduled in the node?